### PR TITLE
fix some typo issue

### PR DESCRIPTION
**Pull request by KeefeL**
## Description

Fix several typo issues.

## Rationale

Make the documents more readable.

### CODE REVIEW

File: docs/guide/greenfield-blockchain/run-node/become-validator.md [hunk 1]
```diff
@@ -28,9 +28,13 @@ Follow the instructions [here to set up a full node](./run-node.md).
...
 :::warning
-The current key generation and storage procedures are not very secure. It is highly recommended to implement a more robust method, particularly when dealing with keys like the `delegator` and `operator` keys.
+The current key generation and storage procedures are not very secure. It is highly recommended to implement a more
+robust method, particularly when dealing with keys like the `delegator` and `operator` keys.
 
-For enhanced security and best practices, the usage of the `coldwallet` and `MPC` wallet is strongly encouraged. These wallets are fully compatible with the `gnfd-tx-sender` tool, and for further information and guidance, please consult the **[gnfd-tx-sender documentation](https://gnfd-tx-sender.nodereal.io/)**.
+For enhanced security and best practices, the usage of the `Cold Wallet` and `MPC Wallet` is strongly encouraged.
+These wallets are fully compatible with the `gnfd-tx-sender` tool, you can use `gnfd` with `--print-eip712-msg-type`
+flags to generate inputs data of each transaction, and then go to [gnfd-tx-sender](https://gnfd-tx-sender.nodereal.io/)
+`Custom Tx` page to send transactions.
```
Comment (zjubfd) [inline docs/guide/greenfield-blockchain/run-node/become-validator.md:37]: I think the tx sender page should have some explain about how to generate custom tx use our CMD

File: docs/guide/storage-provider/run-book/join-SP-network.md [hunk 2]
```diff
@@ -55,14 +55,18 @@ values={[
 
 #### Hot Wallet Manual
 
-You can use the gnfd command to send the create storage provider transaction directly if you import your funding private key into the Keystore. However, this is not safe enough to use a hot wallet for Mainnet, so you can follow the [Hardware Wallet Manual](#hardware-wallet-manual) when you use a hardware wallet.
+You can use the gnfd command to send the create storage provider transaction directly if you import your funding private
```
Comment (zjubfd) [inline docs/guide/storage-provider/run-book/join-SP-network.md:58]: `You can use the ﻿gnfd command to directly send the transaction for creating a storage provider. To do this,  please import the private key of the funding account into the Keystore.`

File: docs/guide/storage-provider/run-book/join-SP-network.md [hunk 3]
```diff
@@ -55,14 +55,18 @@ values={[
...
+key into the Keystore. However, this is not safe enough to use a hot wallet for Mainnet, so you can follow the
```
Comment (zjubfd) [inline docs/guide/storage-provider/run-book/join-SP-network.md:59]:
```
However, it is not safe to use a hot wallet for Mainnet. Instead, you should refer to the Hardware Wallet Manual for instructions on using a hardware wallet.
```

File: docs/guide/greenfield-blockchain/run-node/become-validator.md (hunk 1 above)
Comment (KeefeL) [reply docs/guide/greenfield-blockchain/run-node/become-validator.md]: Sure, added.

File: docs/guide/storage-provider/run-book/join-SP-network.md (hunk 2 above)
Comment (KeefeL) [reply docs/guide/storage-provider/run-book/join-SP-network.md]: fixed

File: docs/guide/storage-provider/run-book/join-SP-network.md (hunk 3 above)
Comment (KeefeL) [reply docs/guide/storage-provider/run-book/join-SP-network.md]: fixed

**Review by zjubfd — APPROVED** [review]

**Pull request merged**